cloud-sql/postgres/activerecord/config/database.yml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START cloud_sql_postgres_activerecord_create]
+development:
+  adapter: postgresql
+  # [START_EXCLUDE]
+  # [START cloud_sql_postgres_activerecord_limit]
+  # 'pool' is the maximum number of permanent connections to keep.
+  pool: 5
+  # [END cloud_sql_postgres_activerecord_limit]
+  # [START cloud_sql_postgres_activerecord_timeout]
+  # 'timeout' is the maximum number of seconds to wait when retrieving a
+  # new connection from the pool. After the specified amount of time, an
+  # ActiveRecord::ConnectionTimeoutError will be raised.
+  timeout: 5000
+  # [END cloud_sql_postgres_activerecord_timeout]
+
+  # [START cloud_sql_postgres_activerecord_backoff]
+  # ActiveRecord automatically uses delays between failed connection attempts,
+  # but provides no arguments for configuration.
+  # [END cloud_sql_postgres_activerecord_backoff]
+  
+  # [END_EXCLUDE]
+  username: <%= ENV["PG_USERNAME"] %>
+  password: <%= ENV["PG_PASSWORD"] %>
+  database: <%= ENV.fetch("PG_DATABASE") { "vote_development" } %>
+  # If connecting via unix domain socket, specify the socket path as host
+  host:  '/cloudsql/<%= ENV["INSTANCE_CONNECTION_NAME"] %>'
+  # If connecting via TCP, enter the IP and port instead
+  # host: '127.0.0.1'
```

Comment:
I know we use a different IP down below, but we should probably clarify here as well. User's will only see this portion from c.g.c.

Reply:
fixed